platform/openide.loaders/src/org/openide/loaders/DataFolder.java

```diff
@@ -835,10 +835,11 @@ protected DataObject handleCreateFromTemplate (
         DataFolder newFolder = (DataFolder)super.handleCreateFromTemplate (f, name);
         Enumeration<DataObject> en = children ();
 
+        Map<String, Object> params = CreateAction.getCallParameters(null);
```

Comment:
Good, this is consistent with `MultiDataObject.create` ... would it be appropriate to also call `FileBuilder.createFromTemplate` on the folder's FileObject ifself ? If no handler is interested (`CreateFromTemplateHandler.accept`) then creation can proceed recursively; otherwise, handlers can be defined for custom folder processing.

Reply:
As you wish: 76105b3